src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -975,7 +975,135 @@ function setGaugeStyleSheet(gaugeName, css, cssback, csstext)
   setLabelStyleSheet(gaugeName .. "_text", csstext or "")
 end
 
+-- https://wiki.mudlet.org/w/Manual:Lua_Functions#getHTMLspan
+-- used by xEcho for creating formatting span tags for labels
+-- fmt is a table of format options as returned by getTextFormat
+function getHTMLspan(fmt)
```

Comment:
How about `getHTMLformat` to complement `getTextFormat`?

Reply:
hmm, I like changing the other function to getLabelFormat, as it returns the same style table as getTextFormat and they complement each other largely, but getHTMLspan returns a string and I'm not sure that quite fits in the same way

Reply:
Yeah that is true, but to me the return has less of importance - getHTMLspan will essentially return the html-ised version of getTextFormat right, so that makes it a nice fit for getHTMLformat(getTextFormat()) for example.

The `span` bit will be lost on people as it's an underlying detail and if they're using c/d/hechoes for labels, chances are they don't know or want to know what a span is.

@keneanung @Kebap could you add what you think is the natural way for you guys here?

Reply:
Upon further reflection I don't hate this idea. I honestly doubt the people using getHTML~~span~~format directly would be the same as those using c/d/hecho for Labels. If anything this would be more of an advance workaround for someone who want more compact html for a format than using "<b><i>" would produce in c/d/hecho. 

This will likely see the majority of its use by c/d/hecho itself and could potentially be undocumented or local but it seems a shame to not expose it for those people who might find it useful.